main.go

```diff
@@ -1,13 +1,21 @@
 package main
 
 import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 
 	"github.com/castai/terraform-provider-castai/castai"
 )
 
+var (
+	commit  = "undefined"
```

Comment:
is this intended, commit is not used anywhere

Reply:
it is set during release, kept it for visibility